.sync.yml

```diff
@@ -11,19 +11,19 @@ Gemfile:
   optional:
     ":development":
       - gem: beaker
-        version: '~> 4.30'
+        version: '~> 5.0'
         from_env: BEAKER_VERSION
       - gem: beaker-abs
         from_env: BEAKER_ABS_VERSION
-        version: '~> 0.9'
+        version: '~> 1.0'
       - gem: beaker-pe
```

Comment:
Need to figure out why `beaker-puppet_install_helper` 0.9.4 is getting installed, yet when this is changed to `beaker-pe -> 3.0`, then we get `beaker-puppet_install_helpers 0.9.8` which triggers the pe/aio bug.

Reply:
beaker-puppet_install_helper 0.9.5 - 0.9.8 all require beaker 4.0 and beaker-pe 2.0:

https://github.com/puppetlabs/beaker-puppet_install_helper/blob/41c355b1f83fde562dd5bd6961e2e4eb8b81421c/beaker-puppet_install_helper.gemspec#L24 

So bundler backtracks to 0.9.4:

https://github.com/puppetlabs/beaker-puppet_install_helper/blob/7591ccb14b3ac9efedcdbb03e26b93902d9c0481/beaker-puppet_install_helper.gemspec#L20

and that version doesn't depend on beaker-pe at all, leading to the bug.